.rubocop.yml

```diff
@@ -0,0 +1,31 @@
+inherit_gem:
+  scc-codestyle:
+    - default.yml
+
+inherit_from: .rubocop_todo.yml
```

Comment:
we also have yast styleguide:

usage: https://github.com/yast/yast-bootloader/blob/master/.rubocop.yml#L3

how it looks like: https://github.com/yast/yast-devtools/blob/master/ytools/y2tool/rubocop_yast_style.yml

Reply:
Yep, I've seen it before. But since we've got our own code style, I've went with it (I've had to disable some peskier check in in `.rubocop_todo.yml`, but otherwise it seems to work pretty well).